### PR TITLE
fix(badge-type): #MBA-69 make user-lists more responsive.

### PR DESCRIPTION
**Pull request by qpic-cgi**
## Describe your changes
Faire en sorte que les éléments de la liste des utilisateurs restent lisibles, et ce, de manière responsive.

## Checklist tests
- Aller sur la page d'un type de badge qui a beaucoup d'activité (badge reçus / badge assignés à d'autre users).
- Tester le responsive
- Constater que le nom des utilisateur reste lisible.

## Issue ticket number and link
[Jira - MBA-69](https://jira.support-ent.fr/browse/MBA-69)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/resources/public/sass/global/_mixins.scss [hunk 1]
```diff
@@ -23,6 +23,12 @@ $minibadge-shadow-light: rgba(214, 214, 214, 0.3);
   }
 }
 
+@mixin tablet {
+  @media (min-width: 769px) and (max-width:992px) {
+    @content;
```
Comment (sjollois) [inline src/main/resources/public/sass/global/_mixins.scss:28]: ça fait quoi @content ?

**Review by sjollois — APPROVED** [review]

File: src/main/resources/public/sass/global/_mixins.scss (hunk 1 above)
Comment (qpic-cgi) [reply src/main/resources/public/sass/global/_mixins.scss:28]: ça applique le contenu de la surcharge de ta mixin:

https://thoughtbot.com/blog/sasss-content-directive

Comment (Sofianebernoussi) [reply src/main/resources/public/sass/global/_mixins.scss:28]: ca va recup le contenu par exemple :   

```
@include tablet {
      flex-direction: column;
}
```

 il recup flex-direction

**Review by Sofianebernoussi — APPROVED** [review]

File: src/main/resources/public/sass/global/_mixins.scss (hunk 1 above)
Comment (qpic-cgi) [reply src/main/resources/public/sass/global/_mixins.scss:28]: trop lent, petit scarabée ! 🪲

**Pull request merged**